doc-processor-gradle-plugin/src/main/kotlin/nl/jolanrensen/docProcessor/gradle/gradleLogging.kt

```diff
@@ -0,0 +1,54 @@
+package nl.jolanrensen.docProcessor.gradle
+
+import mu.KLogger
+import org.gradle.api.logging.Logger
+
+fun KLogger.lifecycle(msg: () -> Any?) {
```

Comment:
Wow, very cool new logger, I'd like this

Reply:
Haha, this is just the Gradle lifecycle-level extension that was missing from KLogger. It's harder for a user to configure logs now, but at least they now operate similar to other gradle plugins